### DISCUSSION TITLE
Using custom ID3 tags in custom fields

### DISCUSSION THREAD

**Discussion by ncfernan** [AzuraCast/AzuraCast#4232] [Q&A] [answered]
My MP3 collection has a custom ID3 tag named `year_name` that I would like to publish in the `now_playing` data.

My custom field in Azuracast is also named `year_name` for convenience, but I cannot link this custom field to the custom tag because the list of available tags is static, it does not allow me to specify which tag must be used to fill the custom field besides those in the list.

![image](https://user-images.githubusercontent.com/10008844/120044800-ed52b000-bfe4-11eb-993e-5511249ad513.png)

```
    "now_playing": {
        "elapsed": 32,
        "remaining": 241,
        "sh_id": 77191,
        "played_at": 1622237333,
        "duration": 273,
        "playlist": "",
        "streamer": "",
        "is_request": false,
        "song": {
            "id": "bb392caa886189acc5256b3fbed0271f",
            "text": "38 Special - Caught up in you",
            "artist": "38 Special",
            "title": "Caught up in you",
            "album": "The Very Best of the A&M Years (1977-1988)",
            "genre": "Southern Rock",
            "lyrics": "",
            "art": "https://stream.ncfradio.com/api/station/4/art/b7d4e7099b56972be0743e79-1622237138.jpg",
            "custom_fields": {
                "year_name": null
            }
        }
    },

``` 

Q: How do I link my custom field named `year_name` to the ID3 tag named `year_name`?

**Comment by BusterNeece** [comment, marked as answer]
@ncfernan If you're not using one of the standard Id3 tags that we provide for possible auto-assignment, it would require some custom code injected directly into the AzuraCast application to achieve; it may be better suited as a [feature request](https://features.azuracast.com/).

**Reply by ncfernan** [reply]
I just opened a feature request. Thanks
